drivers/clock_control/nrf_power_clock.c

```diff
@@ -254,6 +259,7 @@ static int _k32src_start(struct device *dev, clock_control_subsys_t sub_system)
 			__ASSERT_NO_MSG(err == -EINPROGRESS);
 		}
 	}
+#endif /* NRF_CLOCK_HAS_CALIBRATION */
```

Comment:
Because when the corresponding `#if` is only a couple of lines above and it can be easily identified without the help of such comment, there is no need to add one.
In places like this:
```
#if NRF_CLOCK_HAS_CALIBRATION
	u8_t ctto, done;
	struct device *dev = arg;
#endif
#if defined(CONFIG_USB) && defined(CONFIG_SOC_NRF52840)
	bool usb_detected, usb_pwr_rdy, usb_removed;
#endif
```
adding of such comments after `#endif` does not necessarily improve the code readability:
```
#if NRF_CLOCK_HAS_CALIBRATION
	u8_t ctto, done;
	struct device *dev = arg;
#endif /* NRF_CLOCK_HAS_CALIBRATION */
#if defined(CONFIG_USB) && defined(CONFIG_SOC_NRF52840)
	bool usb_detected, usb_pwr_rdy, usb_removed;
#endif /* defined(CONFIG_USB) && defined(CONFIG_SOC_NRF52840) */
```